lib/actions/term-groups.js

```diff
@@ -171,3 +173,19 @@ export function exitActiveTermGroup() {
     });
   };
 }
+
+export function requestPaneMaximizedStateToggle() {
+  return (dispatch, getState) => {
+    dispatch({
+      type: TERM_GROUP_TOGGLE_MAXIMIZED,
+      effect() {
+        const {sessions, termGroups} = getState();
+        const {uid} = findBySession(termGroups, sessions.activeUid);
+        dispatch({
```

Comment:
Finding the active TermGroup to maximize should be done on the reducer, I think.
Two actions for this feature seems to much no ?

Implementation of this feature should be like moveToNextPane() action, imo.